templates/templates/preview/model_plain_text.template

```diff
@@ -0,0 +1,96 @@
+{%- if model is defined -%}Model name: {{ model.name }}
+            Executable:  
+            {%- for param in model.run_settings.exe %}
+                {{ param }}
+            {%- endfor %}
+            Executable Arguments:
+            {%- for param in model.run_settings.exe_args %}
+                {{ param }}
+            {%- endfor %}
+            {%- if model.batch_settings -%}
+            Batch Launch: True
+            {% else %}
+            Batch Launch: False
+            {%- endif -%} 
+            {%- if model.params %}
+            Model parameters: 
+            {%- for param, value in model.params.items() %}
+                {{ param }}: {{ value }}
+            {%- endfor -%}
+            {% endif %}
+            {%- if model.files %}
+            {%- if model.files.tagged %}
+            Tagged Files for model configuration: 
+                {%- for tagged in model.files.tagged %}
+                {{ tagged }} -> 
+                {{ model.path }}
+                {%- endfor %}
+            {%- endif -%}
+            {%- if model.files.copy %}
+            Copy files: 
+                {%- for copy in model.files.copy %}
+                {{ copy }} ->
```

Comment:
not sure but did you want newlines between these `source -> target` things? Multiple copies might be weird looking in output, e.g.

```
aaa ->
bbb
ccc ->
ddd
eee ->
fff
```
vs 
```
aaa -> bbb
ccc -> ddd
eee -> fff
```

Reply:
left as they because readability it easier this way